language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -224,7 +224,19 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
             .collect();
         all_structs.append(&mut local_structs);
 
-        debug!(target: "structs", "{}", self.dump_all_structs(&all_structs, false));
+        let mut all: Vec<(mm::StructEnv, Vec<mty::Type>)> = Vec::new();
```

Comment:
Can we plz replace a generic `all` with more descriptive name?
If it is in a tight loop then a short non-telling name is ok, but here it is used many times later and also used in the comments, so it is difficult to highlight it in a plain browser.

Reply:
Done.

Reply:
thx